source/perflib/perflib.h

```diff
@@ -32,6 +32,13 @@ typedef pthread_t thread_t;
 
 # endif
 
+struct thread_arg_st {
+    void (*func)(size_t num);
+    size_t num;
+};
+
+int perflib_run_thread(thread_t *t, struct thread_arg_st *arg);
+int perflib_wait_for_thread(thread_t thread);
```

Comment:
not sure I understand the addition of this export.  Isn't using perflib_run_thread equivalent to calling perflib_run_multi_thread_test with one as the thread count?

Reply:
The thing is the `perflib_run_multi_thread_test()` spawns thread(s) and then keeps waiting for those thread(s) to finish. I really need portable `pthread_create()` and `pthread_wait()` equivalent as I'd like to spawn server thread and keep main thread running to act as a client.